modules/Microsoft.Storage/storageAccounts/queueServices/readme.md

```diff
@@ -35,11 +35,10 @@ This module can be used to deploy a file share service into a storage account.
 | `diagnosticLogCategoriesToEnable` | array | `[allLogs]` | `[allLogs, StorageDelete, StorageRead, StorageWrite]` | The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. |
 | `diagnosticLogsRetentionInDays` | int | `365` |  | Specifies the number of days that logs will be kept for; a value of 0 will retain data indefinitely. |
 | `diagnosticMetricsToEnable` | array | `[Transaction]` | `[Transaction]` | The name of metrics that will be streamed. |
-| `diagnosticSettingsName` | string | `[format('{0}-diagnosticSettings', parameters('name'))]` |  | The name of the diagnostic setting, if deployed. |
+| `diagnosticSettingsName` | string | `'default-diagnosticSettings'` |  | The name of the diagnostic setting, if deployed. |
```

Comment:
I guess this should be updated as well?

Reply:
good point. Done

Reply:
This will become real interesting for the Rest2Carml automation :D But I think by now with some of the constructs we have (like the necessary use of union) we cannot use it to update much anymore anyways :o